src/renderer/components/jotz.js

```diff
@@ -52,37 +52,16 @@ var Jotz = React.createClass({
     this.setState(newState);
   },
 
-  changeView: function(newView) {
+  swapView: function(newView, note) {
     var newState = React.addons.update(this.state, {
       jotzState: {
-        view: {
-          $set: newView
-        }
+        view: { $set: newView },
+        currentNote: { $set: note ? note : null }
```

Comment:
In keeping with the styleguide and not using ternary operators, can we change this to be defined before newState is defined?

```javascript
var note = note || null;
```